### PR TITLE
implement get assets tied to a ticket

### PR DESCRIPTION
**Pull request by ktran0918**
The implementation is lines 1044 - 1063--getAssetsFromTicket()

The other changes are from prettier's formatting.

### CODE REVIEW

File: tdapi.js [hunk 1]
```diff
@@ -1038,15 +1041,35 @@ TDAPI.prototype.addAssetToTicket = function(id, ticketId) {
     .catch(handleError);
 };
 
+/**
+ * Get assets associated with a ticket
+ * The endpoint also returns additional configuration items
+ * @param {Number} [appId]  - The associated application ID to get attributes for.
```
Comment (LivVitale) [inline tdapi.js]:
```suggestion
 * @param {Number} appId  - The associated application ID to get attributes for.
```

Comment (ktran0918) [reply tdapi.js]: It's edited as suggested.

**Comment by ktran0918** [comment]
Can you look over the docs to see if they were generated correctly? Command run:
```
# jsdoc globally installed
jsdoc -c ./jsdoc.conf.json
```

**Comment by LivVitale** [comment]
> Can you look over the docs to see if they were generated correctly? Command run:
> 
> ```
> # jsdoc globally installed
> jsdoc -c ./jsdoc.conf.json
> ```
> 
> Thanks

@ktran0918 You can actually generate the docs by running `npm run build-docs` (which just runs the command you mentioned, but with a copy of jsdoc local to the project instead of globally-installed). Docs look good to me! Feel free to merge. :]

**Pull request merged**